src/requirements.yml

```diff
@@ -5,6 +5,10 @@
   name: amazon_ssm_agent
 - src: https://github.com/cisagov/ansible-role-assessment-tool
   name: assessment_tool
+- src: https://github.com/cisagov/ansible-role-automated-security-updates
+  name: automated_security_updates
+- src: https://github.com/cisagov/ansible-role-backports
+  name: backports
```

Comment:
This can be deleted:
```suggestion
```

Reply:
I'd prefer to let that change trickle down from [upstream](https://github.com/cisagov/skeleton-packer/pull/191).

Reply:
My bad - I thought we had already merged that upstream change in.

Reply:
@jsf9k Looking at the diff this was re-added when merging in from the skeleton since it has yet to be removed there. Please see the previous commit https://github.com/cisagov/teamserver-packer/commit/14b5c2815aa939ee714e776f473c20298120cabc that removed this requirement.

Reply:
Hmmm...@mcdonnnj is correct.  Please see commit 61bf9b1.